tests/backup/backup_test.go

```diff
@@ -86,6 +88,508 @@ var _ = Describe("{UserGroupManagement}", func() {
 	})
 })
 
+var _ = Describe("{BkpRstrDiffK8sVerSimultaneousDiffNS}", func() {
```

Comment:
Place this test under `tests/backup/backup_restore_basic_test.go`

Reply:
✅